provisioning/plugins/corim-psa-decoder/ccaplatformconfigid.go

```diff
@@ -33,19 +33,26 @@ func (o *CCAPlatformConfigID) FromMeasurement(m comid.Measurement) error {
 	return nil
 }
 
-func (o CCAPlatformConfigID) MakeRefAttrs(c PSAClassAttributes) (*structpb.Struct, error) {
+func (o CCAPlatformConfigID) GetRefValType() string {
+	return "platform-config"
+}
+
+// MakeRefAttrs is an interface function to populate reference attributes.
+// For PlatformConfig scheme argument is not strictly required, but is required for other
+// usage of the same interface
```

Comment:
this comment should go on the interface method rather than on an implementation of the interface.

Reply:
I have tidied up the comment to only retain the relevant bits here and moved the top line comment to the interface.